test-suite-kotlin/src/test/kotlin/io/micronaut/docs/server/suspend/SuspendController.kt

```diff
@@ -31,8 +31,7 @@ import org.slf4j.MDC
 class SuspendController(
     @Named(TaskExecutors.IO) private val executor: ExecutorService,
     private val suspendService: SuspendService,
-    private val suspendRequestScopedService: SuspendRequestScopedService,
-    private val coroutineTracingDispatcherFactory: HttpCoroutineContextFactory<CoroutineTracingDispatcher>
```

Comment:
`CouroutineTracingDispatcher` was deleted here https://github.com/micronaut-projects/micronaut-tracing/pull/281/files

No test was using it.